pokemongo_bot/cell_workers/soft_ban_worker.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from pokemongo_bot.cell_workers.utils import distance
+from pokemongo_bot.worker_result import WorkerResult
+from pokemongo_bot.cell_workers import SeenFortWorker
+from pokemongo_bot.cell_workers import MoveToFortWorker
+
+
+class SoftBanWorker(object):
+    def __init__(self, bot):
+        self.running = False
+        self.bot = bot
+    def work(self):
+       if self.bot.softbanned == True:
+           nearest_fort = self.nearest_pokestop()
+           MoveToFortWorker(nearest_fort, self.bot).work()
+           SeenFortWorker(nearest_fort, self.bot).work()
+
+           if self.bot.softbanned == True:
+               return WorkerResult.RUNNING
```

Comment:
This will keep anything after it from running in the tick. This means that NO other worker needs to know about whether we are soft banned. They should not have any check in it at all.